makefiles/tests/tests.inc.mk

```diff
@@ -74,7 +74,7 @@ test-with-config/check-config:
 # this target only makes sense if an ELFFILE is actually created, thus guard by
 # RIOTNOLINK="".
 ifeq (,$(RIOTNOLINK))
-test-input-hash: $(TESTS) $(TESTS_WITH_CONFIG) $(TESTS_AS_ROOT) $(ELFFILE) $(TEST_EXTRA_FILES)
+test-input-hash: $(TESTS) $(TESTS_WITH_CONFIG) $(TESTS_AS_ROOT) $(BINFILE) $(TEST_EXTRA_FILES)
```

Comment:
If I'm right, `BINFILE` is not built in CI: https://github.com/RIOT-OS/RIOT/blob/55b57e1a75a1e447086161012bee9ff499474d96/Makefile.include#L689-L695

Reply:
Hm, I tried it for AVR, only the `FLASHFILE` (`.hex`) is generated when `RIOT_CI_BUILD` is set.